### PR TITLE
Update Availability Zones Information 

### PR DESCRIPTION
**Pull request by MarkvanMents**
Architecture is not correct as it has been changed to reduce latency on database calls.

https://mendix.atlassian.net/browse/TW-817

### CODE REVIEW

File: content/evaluation-guide/enterprise-capabilities/cloud-architecture.md [hunk 1]
```diff
@@ -48,7 +48,9 @@ The Mendix Cloud offers high availability for all app environments, ensuring zer
 
 ## 5 How Does Mendix Provide Disaster Recovery? {#cloud-dr}
 
-Mendix Cloud architecture automatically applies failover between availability zones, so database containers and file storage buckets are automatically replicated across multiple availability zones. All backups are copied to other data centers, but the data will not leave the region, should your company require this.
+Mendix Cloud architecture keeps runtime containers, database containers, and file storage buckets in the same availability zone (AZ) to minimize latency in database operations. It automatically applies failover to a geographically separate AZ if the current AZ fails.  New copies of your app will be started automatically in the new AZ. File storage buckets are automatically replicated by AWS across multiple AZs and, with Mendix high-availability options, the  database containers can also be replicated automatically through streaming – or you can restore a backup to the new AZ.
```
Comment (pommi) [inline content/evaluation-guide/enterprise-capabilities/cloud-architecture.md]: I would suggest to remove ", and file storage buckets".
"high-availability" is officially called "Fallback" on our pricelist.

File: content/evaluation-guide/enterprise-capabilities/cloud-architecture.md [hunk 2]
```diff
@@ -48,7 +48,9 @@ The Mendix Cloud offers high availability for all app environments, ensuring zer
...
+
+All backups are copied to other data centers, but the data will not leave the region, should your company require this.
```
Comment (pommi) [inline content/evaluation-guide/enterprise-capabilities/cloud-architecture.md]: This should be: "All backups are copied to another region, but the data will not leave the continent, should your company require this."

**Comment by MarkvanMents** [comment]
Email sent to Tom and Hans to review PR for accuracy, messaging, and terminology.
Will make requested changes once I know if further changes may be required after review.

File: content/evaluation-guide/enterprise-capabilities/cloud-architecture.md (hunk 1 above)
Comment (MarkvanMents) [reply content/evaluation-guide/enterprise-capabilities/cloud-architecture.md]: Changed

File: content/evaluation-guide/enterprise-capabilities/cloud-architecture.md (hunk 2 above)
Comment (MarkvanMents) [reply content/evaluation-guide/enterprise-capabilities/cloud-architecture.md]: Changed

**Comment by MarkvanMents** [comment]
Approved by Tom de Groot.

Now approaching UX to have the images improved.

**Pull request merged**